src/xtgeo/grid3d/grid.py

```diff
@@ -920,17 +921,22 @@ def from_file(
 
         Example::
 
-            >>> myfile = "../../testdata/Zone/gullfaks.roff"
             >>> xg = Grid()
-            >>> xg.from_file(myfile, fformat="roff")
+            >>> xg.from_file(reek_dir + "/REEK.EGRID", fformat="egrid")
+            Grid ... filesrc='.../REEK.EGRID'
             >>> # or shorter:
             >>> xg = Grid(myfile)  # will guess the file format
 
         Example using "eclipserun"::
 
-            >>> mycase = "ECL"  # meaning ECL.EGRID, ECL.INIT, ECL.UNRST
+            >>> mycase = "REEK"  # meaning REEK.EGRID, REEK.INIT, REEK.UNRST
             >>> xg = Grid()
-            >>> xg.from_file(mycase, fformat="eclipserun", initprops="all")
+            >>> xg.from_file(
+            ...     reek_dir + "/" + mycase,
+            ...     fformat="eclipserun",
```

Comment:
Also to be deprecated

Reply:
True, but it is in the documentation of a deprecated function so that is to be expected :)